src/dotnet/APIView/APIView/Model/TreeTokenCodeFile.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using APIView;
+using csharp_api_parser.TreeToken;
+using System;
+using System.Collections.Generic;
+
+namespace ApiView
+{
+    public class TreeTokenCodeFile
+    {
+        private string _versionString;
+
+        [Obsolete("This is only for back compat, VersionString should be used")]
+        public int Version { get; set; }
```

Comment:
This is a new code file format. why do we need this? Previous CodeFile format had this property since some of the original JSON file uploaded had this property so it was retained instead of removing it from class. I don't think we need it if new parser is not going to set it.

Reply:
I have Removed TreeTokenCodeFile